### DISCUSSION TITLE
Menu bar space

### DISCUSSION THREAD

**Discussion by sevoc** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#844] [General]
Is it possible to reduce the space between Battery Icon/Percentage and other icons in Menu bar? Maybe half of what it is now? On latest Macs with notch every gained inch is valuable. Thanks!

**Comment by MatthiasKerbl** [comment]
Hi @sevoc,

Yes, that is possible. Check out the mac app Bartender.
Furthermore, we are also working on reducing the spacing of the AlDente menu bar icon.

Cheers,
Matthias

**Reply by sevoc** [reply]
Thanks, Matthias!
